near-sdk/src/test_utils/context.rs

```diff
@@ -0,0 +1,87 @@
+use crate::test_utils::test_env::*;
+use crate::{AccountId, Balance, BlockHeight, EpochHeight, PublicKey, VMContext};
+
+/// Simple VMContext builder that allows to quickly create custom context in tests.
+pub struct VMContextBuilder {
+    pub context: VMContext,
+}
+
+#[allow(dead_code)]
+impl VMContextBuilder {
+    pub fn new() -> Self {
+        Self {
+            context: VMContext {
+                current_account_id: alice(),
+                signer_account_id: bob(),
+                signer_account_pk: vec![0u8; 32],
+                predecessor_account_id: bob(),
+                input: vec![],
+                block_index: 0,
+                block_timestamp: 0,
+                epoch_height: 0,
+                account_balance: 10u128.pow(26),
+                account_locked_balance: 0,
+                storage_usage: 1024 * 300,
+                attached_deposit: 0,
+                prepaid_gas: 300 * 10u64.pow(12),
+                random_seed: vec![0u8; 32],
+                is_view: false,
+                output_data_receivers: vec![],
+            },
+        }
+    }
+
+    pub fn current_account_id(mut self, account_id: AccountId) -> Self {
```

Comment:
why not take `&mut self` and return `&mut Self`?

Reply:
done